src/Compiler/Specification.php

```diff
@@ -27,7 +27,7 @@ public function getCompiled(): array
      *
      * @param bool|string $data
      * @param string $type
-     * @return bool|string
+     * @return bool|int|float|string
```

Comment:
To fix following Psalm error, using the `@return bool|int|float|string` annotation:

```
RROR: InvalidReturnType - src/Compiler/Specification.php:30:16 - The declared return type 'bool|float|string' for Mill\Compiler\Specification::convertSampleDataToCompatibleDataType is incorrect, got 'scalar' (see https://psalm.dev/011)
     * @return bool|float|string

```